internal/storage/v2/elasticsearch/tracestore/to_dbmodel.go

```diff
@@ -79,14 +79,23 @@ func resourceToDbProcess(resource pcommon.Resource) dbmodel.Process {
 		process.ServiceName = noServiceName
 		return process
 	}
+	var serviceName string
 	tags := make([]dbmodel.KeyValue, 0, attrs.Len())
 	for key, attr := range attrs.All() {
-		if key == conventions.AttributeServiceName {
-			process.ServiceName = attr.AsString()
-			continue
+		switch key {
+		case conventions.ServiceNameKey:
+			serviceName = attr.AsString()
+		case conventions.AttributeOtelScopeName:
+			tags = append(tags, attributeToDbTag(key, attr))
+		default:
```

Comment:
this changes are not needed, please restore the original code. You only need to change one line in it.